test/tests/ver-1g/tests

```diff
@@ -15,4 +15,28 @@
     requirement = 'The system shall be able to model a chemical reaction between two species with different concentrations and calculate the concentrations of reactants and product as a function of time'
     verification = 'ver-1g.md'
   []
+  [binary_reaction_same_concentrations_csv_diff]
+    type = CSVDiff
+    input = ver-1g_same_conc.i
+    should_execute = False # this test relies on the output files from binary_reaction_same_concentrations, so it shouldn't be run twice
+    csvdiff = ver-1g_same_conc_out.csv
+    requirement = 'The system shall be able to model a chemical reaction between two species with the same concentrations and calculate the concentrations of reactants and product as a function of time, with the fine mesh and time step required to match the analytical solution to generate CSV data for use in comparisons with the analytic solution over time.'
```

Comment:
I don't think you are using a different mesh than the exodus test, are you? If the input file is fast enough to run as a test AND to match the analytical solution, then:
- you can remove the exodiff tests since they are replaced by your new tests.
- remove the exodus output from the input file.
- you should also remove the part of the requirement about fine mesh and time step.

Reply:
@ChaitanyaBhave-e please take note of the edit.